src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -122,58 +122,55 @@ function adjustStudentsPanelView(panelHeading, panelCollapse, isDataHandsontable
 }
 
 /**
- * Expands "Existing students" panel and loads existing students' data (if spreadsheet is not empty)
- * into the spreadsheet interface. Spreadsheet interface would be shown after expansion.
+ * Expands "Existing students"/"New students" panel. Spreadsheet interface would be shown after expansion.
+ * If it is the "New students" panel, its affiliated buttons will also be shown.
+ * If it is the "Existing students" panel, an AJAX request would be called to load existing
+ * students' data into the spreadsheet interface (if spreadsheet is not empty).
  * The panel will be collapsed otherwise if the spreadsheet interface is already shown.
  */
-function expandCollapseExistingStudentsPanel() {
-    const panelHeading = $(this);
-    const panelCollapse = panelHeading.parent().children('.panel-collapse');
-
-    const ajaxPreloadImage = panelHeading.parent().find('#ajax-preload-image');
-    const ajaxStatusMessage = panelHeading.parent().find('.ajax-status-message');
-    ajaxStatusMessage.hide(); // hide any status message from the previous state
 
-    // perform AJAX only if existing students' spreadsheet is empty
-    if (getSpreadsheetLength(dataHandsontable.getData()) === 0) {
-        getAjaxStudentList(ajaxPreloadImage)
-                .then((data) => {
-                    ajaxPreloadImage.toggle();
-                    if (data.students.length === 0) {
+function studentsPanelClickHandler(currentSpreadsheet) {
```

Comment:
The right argument name is `event`, from Jquery's documentation https://api.jquery.com/on/